src/main/java/com/minecolonies/coremod/client/gui/WindowHutBuilder.java

```diff
@@ -18,12 +29,31 @@
     /**
      * The builders gui file.
      */
-    private static final String HUT_BUILDER_RESOURCE_SUFFIX = ":gui/windowhutbuilder.xml";
-    private static final String LIST_RESOURCES              = "resources";
-    private static final String PAGE_RESOURCES              = "resourceActions";
-    private static final String VIEW_PAGES                  = "pages";
+    private static final String HUT_BUILDER_RESOURCE_SUFFIX        = ":gui/windowhutbuilder.xml";
+    private static final String LIST_RESOURCES                     = "resources";
+    private static final String PAGE_RESOURCES                     = "resourceActions";
+    private static final String VIEW_PAGES                         = "pages";
+    private static final String RESOURCE_NAME                      = "resourceName";
+    private static final String RESOURCE_AVAILABLE_NEEDED          = "resourceAvailableNeeded";
+    private static final String RESOURCE_ADD                       = "resourceAdd";
+    private static final String RESOURCE_ID                        = "resourceId";
+    private static final int RESOURCE_ID_POSITION                  = 3;
+    private static final String RESOURCE_QUANTITY_MISSING          = "resourceQuantity";
+    private static final int RESOURCE_QUANTITY_MISSING_POSITION    = 4;
+
+    private static final int red       = Color.getByName("red",0);
+    private static final int darkgreen = Color.getByName("darkgreen",0);
+    private static final int black     = Color.getByName("black",0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00115)